docs/installation.md

```diff
@@ -14,6 +14,24 @@ pip install improv --no-binary pyzmq
 to build `pyzmq` from source if you're running into ZMQ errors.
 ````
 
+## Required dependencies
+
+### Redis
+
+_improv_ uses Redis, an in-memory datastore, to hold data to be communicated between actors. _improv_ has been tested with Redis server version 7.2.4. Please refer to the instructions below for your operating system:
+
+#### macOS
+A compatible version of Redis can be installed via Homebrew:
```

Comment:
Can you add a link to Homebrew's page?

Reply:
I think that you should also include an internal link to "Required dependencies" under "Simple installation" so that it says something like
> Once you have the [required dependencies](link-to-section), the simplest way to install _improv_...